README.md

```diff
@@ -194,6 +194,51 @@ E.g. `Foo = [name: tstr, age: uint]` is equivalent to `Foo = [tstr, uint]`.
 
 See [test3_simple](tests/decode/test3_simple/) for CDDL example code.
 
+Introduction to CBOR
+====================
+
+CBOR's format is described well on Wikipedia
+ - https://en.wikipedia.org/wiki/CBOR
+but here's a synopsis:
+
+Encoded CBOR data elements look like this.
+
+```
+| Header             | Value                  | Payload                   |
+| 1 byte             | 0, 1, 2, 4, or 8 bytes | 0 - 2^64-1 bytes/elements |
+| 3 bits | 5 bits    |
+| Major  | Additional|
+| Type   | info      |
+```
```

Comment:
```suggestion
| Header | Value  | Payload |
| --- | --- | --- |
| 1 byte | 0, 1, 2, 4, or 8 bytes | 0 - 2^64-1 bytes/elements |
| 3 bits | 5 bits | |
| Major  | Additional | |
| Type   | info | |
```
Which gets rendered in Markdown as:
| Header | Value  | Payload |
| --- | --- | --- |
| 1 byte | 0, 1, 2, 4, or 8 bytes | 0 - 2^64-1 bytes/elements |
| 3 bits | 5 bits | |
| Major  | Additional | |
| Type   | info | |

Reply:
I want the "Header" and "1 byte" to span 2 columns, which is not possible in markdown AFAIK, which is why I kept it as preformatted ASCII-art. I changed the table slightly now, see if it works or if I should come up with something else.

Reply:
Ah I see, no worries then.